tests/end-to-end/director/fields.go

```diff
@@ -166,3 +166,14 @@ func (fp *gqlFieldsProvider) ForRuntime() string {
 		status {condition timestamp}
 		agentAuth {%s}`, fp.ForAuth())
 }
+
+func (fp *gqlFieldsProvider) ForApplicationLabel() string {
+	return `applicationID
```

Comment:
There's no `applicationID` in Label type

Reply:
done